src/styles/blocks/main.scss

```diff
@@ -0,0 +1,155 @@
+.main {
+  // position: relative;
+  display: flex;
+  flex-wrap: wrap;
+
+  // position: absolute;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  justify-content: center;
+}
+
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
+  width: 100%;
+
+  // margin: 0;
+  top: 110px;
+  justify-content: center;
+}
+
+@media (min-width: 300px) {
+  .cards {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background-color: $white-color;
```

Comment:
The variable `$white-color` is used here but not defined in this file. Ensure that it is defined in a global variables file or within this file to avoid errors.